### ISSUE TITLE
--latency-ms not longer supported in factorio 0.14 experimental

### ISSUE THREAD

**Issue by NochEinKamel** [Bisa/factorio-init#79]
Launching the new experimental 0.14 with the script no longer works as it uses a no longer supported command line option which needs to be removed from the INVOCATION variable.

```
/opt/factorio/bin/x64/factorio --config /opt/factorio/config/config.ini --port 34197 --start-server-load-latest --server-settings /opt/factorio/data/server-settings.json --autosave-interval 5 --autosave-slots 24 --latency-ms 75
unrecognised option '--latency-ms'
```

(And thanks for the script, really helps running a server!)

**Comment by Bisa** [comment]
Thank you for the report, I'll have it removd

**Comment by Bisa** [comment]
fixed in commit 8e2cc53d55c6e0db7bb53484fc21fc820efe4c4c
